### DISCUSSION TITLE
Extended Services Windows

### DISCUSSION THREAD

**Discussion by Yoss27** [community/community#137005] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

GitHub Education

### Body

Donde puedo descargar este fichero para la extensión de servicio en windows, la página sale con error:   

 kie_sandbox_extended_services_windows_0.32.0.exe

![kogito (1)](https://github.com/user-attachments/assets/b17a8c74-f5ad-49f6-90b2-fc69f74266f7)

**Comment by queenofcorgis** [comment]
Hi @Yoss27 , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.

_Please note for future posts:  The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them._
